pom.xml

```diff
@@ -198,51 +198,66 @@ SOFTWARE.
       <scope>test</scope>
     </dependency>
   </dependencies>
-  <build>
-    <testResources>
-      <testResource>
-        <directory>${basedir}/src/test/resources</directory>
-        <filtering>true</filtering>
-      </testResource>
-    </testResources>
-    <plugins>
-      <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <configuration>
-          <finalName>artipie</finalName>
-          <archive>
-            <manifest>
-              <mainClass>com.artipie.VertxMain</mainClass>
-            </manifest>
-          </archive>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
-        </configuration>
-        <executions>
-          <execution>
-            <id>make-assembly</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <artifactId>maven-failsafe-plugin</artifactId>
-        <configuration>
-          <parallel>none</parallel>
-        </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>integration-test</goal>
-              <goal>verify</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
+  <profiles>
+    <profile>
+      <id>qulice</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <testResources>
+          <testResource>
+            <directory>${basedir}/src/test/resources</directory>
+            <filtering>true</filtering>
+          </testResource>
+        </testResources>
+        <plugins>
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
@marceloamadeu based on my interpretation of #175, we should move this to a _separate_ profile instead of including i in `qulice` profile. Besides, it does not make sense to tie this in to the static analysis of the Qulice plugin which is what the qulice profile.

Tl;dr, let's name it something other than `qulice` which is a profile we use for something else.

Reply:
@carlosmiranda I understand your request for change ... Before I do I would like to ask @victornoel's opinion.

Tks

Reply:
@carlosmiranda @marceloamadeu I don't see any reason to have the maven-assembly-plugin in the same profile as qulice, those are two very different things and most certainly not meant to be used for the same reason.

I would think introducing a separate profile should be a good solution.

Note that I have no idea what this assembly file is used for, so maybe @g4s8 can chip in to tell us where the profile should be activated in the CI or in rultor for releases maybe?

Reply:
@carlosmiranda also, why put the tests related things in a profile? I think there were perfects where they were before IMHO.

Reply:
@victornoel yes, you are right, this profile should be enabled with Docker builds in Github actions.

Reply:
@marceloamadeu any updates on this?